src/ui/viewmodels/MemoryViewerViewModel.hh

```diff
@@ -175,6 +175,10 @@ public:
     void AdvanceCursorPage();
     void RetreatCursorPage();
 
+    void SaveToMemViewHistory();
+    void MoveMemViewHistoryBackward();
+    void MoveMemViewHistoryForward();
```

Comment:
`MemView` is redundant in these function names as they're part of the `MemoryViewerViewModel`.